api/RS.API/RS.Data/Logic/UserRepository.cs

```diff
@@ -2,14 +2,27 @@
 using RS.Data;
 using RS.Entity;
 using RS.Entity.Models;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System;
+using RS.ViewModel.User;
 
 namespace RS.Data.Logic
 {
     public class UserRepository: Repository<Users>, IUserRepository
     {
         private readonly RSContext _context;
+        private DbSet<Users> entities;
         public UserRepository(RSContext context) : base(context) {
             this._context = context;
         }
+
+        public Users LoginUser(string username, string password)
+        {
+            Users user = _context.Users.Where(x => x.UserName == username && x.Password == password)
+                                .Select(x => x ).SingleOrDefault();
```

Comment:
use firstordefault instead of singleordefault and remove select statement;
use return direct instead of variable assignment

Reply:
Changes done